addon/components/data-table.hbs

```diff
@@ -34,19 +34,18 @@
           @vertical="middle"
           class="uk-height-large"
         >
-         <UkSpinner @ratio="3" />
+          <UkSpinner @ratio="3" />
         </UkFlex>
       </td>
     </tbody>
   {{else}}
     {{yield
       (hash
-        body=(component "data-table/body" models=this.models)
+        body=(component "data-table/body" models=this.data.value)
```

Comment:
As we are using `ember-resources` now, the data fetching is delayed until this component is loaded. This means, using the `<DataTable as |table|/>` without  the `table.body` call would result in a table which never fetches data. Is this valid (the use-case for this scenario seems rare...) or should we lift the fetching artifically higher in the component tree? @czosel

Reply:
I agree, data-tables without body don't seem to make sense.